api/auth_middleware.go

```diff
@@ -69,7 +67,19 @@ func authMiddleware(sasUC usecases.ServiceAccounts) func(http.Handler) http.Hand
 	}
 }
 
-func buildAuth(header, method, content string) authorizationHeader {
+func buildAuth(authHeader string) (authorizationHeader, error) {
```

Comment:
nit: return (*authorizationHeader, error)? This would lead to the more common `return nil, err` idiom.